src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -322,14 +398,24 @@ temperature_archery <- temperature_archery %>%
   mutate(flag = if_else(p1 >= temp_max | p1 <= temp_min, "Out of bounds", flag),
          p1 = if_else(p1 >= temp_max | p1 <= temp_min, NA_real_, p1))
 
+temperature_archery <- temperature_archery %>%
```

Comment:
Out of range, slope, sd stuff WILL be different across site/parameter types so good job keeping this out of the upstream function :-)